praw/models/reddit/subreddit.py

```diff
@@ -873,6 +877,120 @@ def submit(
 
         return self._reddit.post(API_PATH["submit"], data=data)
 
+    def submit_gallery(
+        self,
+        title,
+        images,
+        flair_id=None,
+        flair_text=None,
+        resubmit=True,
+        send_replies=True,
+        nsfw=False,
+        spoiler=False,
+        collection_id=None,
+        discussion_type=None,
+    ):
+        """Add an image gallery submission to the subreddit.
+
+        :param title: The title of the submission.
+        :param images: The images to post in dict with the following structure:
+            ``{"image_path": "path", "caption": "caption", "outbound_url": "url"}``,
+            only ``"image_path"`` is required.
+        :param collection_id: The UUID of a :class:`.Collection` to add the
+            newly-submitted post to.
+        :param flair_id: The flair template to select (default: None).
+        :param flair_text: If the template's ``flair_text_editable`` value is
+            True, this value will set a custom text (default: None).
+        :param resubmit: When False, an error will occur if the URL has already
```

Comment:
Does this parameter make sense here?

Reply:
Which parameter?

Reply:
resubmit

Reply:
It is a query param that is sent by the browser.

Reply:
I understand, but is it ever actually needed for a gallery submission? That param is used for link submissions to allow one to resubmit a link that has already been submitted. I'm not sure if this would apply in any way for gallery submissions. If it doesn't, then I don't think we should include it.

Reply:
I don't think so, I think I did try without it and I don't think it threw an error. I included it because `submit_image` includes it and I think `submit_video` does as well.

Reply:
Let's pull it out for this addition. We should probably pull it out of the others too.